code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -167,7 +167,8 @@
 			stored_mmi.forceMove(get_turf(owner))
 			stored_mmi = null
 	..()
-	qdel(src)
+	if(!QDELETED(src))
+		qdel(src)
```

Comment:
this finally fixes a long-standing `Destroy` loop that has eluded me when IPC players become nuke ops.